### ISSUE TITLE
[Bug] Missing some icons in Windows Manager Settings

### ISSUE THREAD

**Issue by notuxius** [netrunner-rolling/QA-Suggestions#27]
Right click on any window with system title bar -> More Actions -> Windows Manager Settings -> Window Decorations
![missing-some-icons-in-windows-manager-settings](https://user-images.githubusercontent.com/2657014/28609300-d84d03d0-71ec-11e7-975d-b3a498a6eb24.png)

**Comment by NuLogicSystems** [comment]
This is an upstream plasma bug. You can report it to https://bugs.kde.org/.

**Comment by notuxius** [comment]
Seems like this is Netrunner LnF theme bug - some icons are the same color as the background.
Changing 'Color' for the theme will reveal icons:
![icons-bug-in-netrunner-lnf-theme](https://user-images.githubusercontent.com/2657014/28859434-e4461f82-775f-11e7-87c3-0ee705c0d09c.gif)

**Comment by NuLogicSystems** [comment]
No, it's the breeze Icon theme that has some light coloured icons from the breeze dark theme, so of course choosing a dark color scheme is going to make them show. Once again, this needs to be reported upstream so that the KDE artwork team can fix it.

**Comment by NuLogicSystems** [comment]
OK, it's not a bug, those icons are light because the window decorations theme has a dark blue border, they need to be a light color. this is part of the breeze widget themes design.

**Comment by NuLogicSystems** [comment]
Our Carbon theme is based on oxygen cold, but here is the thing, changing to oxygen or oxygen cold there are dark icons under Titlebar, switching back to carbon they stay dark. Next, switch to the breeze color scheme and these turn to that gray color, switching back to carbon they stay grey. It could be that there is something missing from our Carbon color scheme, I'm not sure since I didn't create it, or it could be a bug in the preview window. However, this doesn't seem to effect actual usability, it's just the preview dialog for Titlebars that is effected, not the actual theme.

**Comment by notuxius** [comment]
They are present on default light theme on Manjaro KDE
![manjaro-window-decorations-present-icons](https://user-images.githubusercontent.com/2657014/28870942-ef422414-778a-11e7-952b-fe2f64f63663.png)

**Comment by NuLogicSystems** [comment]
Like I said, It's just the preview window and does not effect the actual theme, please read my post again. I did not write our carbon color scheme, I only packaged it. However, we have been using our carbon color scheme since KDE 4 and it may need an update since it is based on an older version of Oxygen cold.

PS. I'm a Manjaro and AUR maintainer as well, but I did not write Maia either. Maia is loosely based on Breeze with color changes from Plasma blue to Manjaro green. Breeze also has the same light grey color icons in that preview dialog.

**Comment by notuxius** [comment]
Manjaro KDE Plasma theme looks nice - maybe it can be used as a base for next updated Netrunner theme

**Comment by NuLogicSystems** [comment]
It would still have the same problem with carbon, it's based on breeze.
Carbon needs to be compared to the latest oxygen to see why that preview is not working right, or at all actually.
